frontend-voting/src/components/PageNotFound.tsx

```diff
@@ -1,11 +1,32 @@
 
+import { Link } from "react-router-dom";
+import { FaSadCry } from "react-icons/fa";
+
 const PageNotFound = () => {
   return (
-    <div className="text-white w-screen h-screen flex justify-center items-center">
-      <h1 className="text-6xl font-bold">404</h1>
-      <h1 className="text-2xl font-bold">Page Not Found</h1>
+    <div className="flex flex-col justify-center items-center min-h-screen bg-gradient-to-r from-gray-900 to-gray-700 text-white relative">
+      <div className="absolute inset-0 bg-cover bg-center opacity-20" style={{ backgroundImage: "url('https://your-image-url.com')" }}></div>
+      
+      <div className="relative z-10 flex flex-col justify-center items-center text-center">
+        <FaSadCry className="text-6xl mb-4 animate-bounce" />
+        <h1 className="text-8xl font-extrabold animate-pulse">404</h1>
+        <h2 className="text-3xl font-semibold mt-4">Oh no! Page Not Found</h2>
+        <p className="mt-2 text-lg text-gray-300">
+          The page you are looking for does not exist. But don’t worry, you can find your way back!
+        </p>
+        <Link 
+          to="/" 
+          className="mt-6 px-6 py-3 bg-blue-600 rounded-lg shadow-lg hover:bg-blue-700 transition duration-300 transform hover:scale-105"
```

Comment:
may be you can just change the color of the btn inorder to be in the same ui as that of home page.